src/looprestoration_tmpl_16.rs

```diff
@@ -130,15 +129,19 @@ extern "C" {
 
 pub type pixel = uint16_t;
 pub type coef = int32_t;
-use crate::src::looprestoration::LrEdgeFlags;
 pub type const_left_pixel_row = *const [pixel; 4];
-use crate::src::looprestoration::LooprestorationParams;
 
-use crate::include::common::intops::iclip;
-use crate::src::looprestoration::padding;
-use crate::src::looprestoration::selfguided_filter;
+#[cfg(feature = "asm")]
+cfg_if! {
+    if #[cfg(any(target_arch = "arm", target_arch = "aarch64"))] {
+        use crate::src::looprestoration::LrEdgeFlags;
+        use crate::src::looprestoration::LooprestorationParams;
+    }
+}
```

Comment:
Why's the `#[cfg]` logic done twice here and with `cfg_if!`?

Can it be this?

```suggestion
#[cfg(all(feature = "asm", any(target_arch = "arm", target_arch = "aarch64")))]
use crate::{
    src::looprestoration::LrEdgeFlags,
    src::looprestoration::LooprestorationParams,
};
```

Reply:
Same for 8bpc.

Reply:
Ah yeah, I like your suggested approach better. The only thing is that rustfmt tries to put all the imports on the same line, so I also threw a `#[rustfmt::skip]` on there. I don't love having to do that, but these conditional imports are just to suppress unused warnings, and will go away once deduplication for `looprestoration.rs` is done.

Reply:
That sounds good.  `#[rustfmt::skip]` should be fine for temporary things.